.github/workflows/ci.yml

```diff
@@ -946,8 +946,8 @@ jobs:
           CXX_STANDARD: '14'
         run: |
           ./ci/do_ci.sh cmake.w3c.trace-context.build-server
-          cd $HOME/build/ext/test/w3c_tracecontext_test
-          ./w3c_tracecontext_test &
+          cd $HOME/build/ext/test/w3c_tracecontext_test_server
+          ./w3c_tracecontext_test_server &
```

Comment:
if we want to change it `w3c_tracecontext_http_test_server` would be better, to emphasize this is actually HTTP server. @ThomsonTan  - what you suggest ?

Reply:
I agree with `w3c_tracecontext_http_test_server`, it makes it clearer that the way to talk to it is with curl.

Reply:
+1.